appengine_config.py

```diff
@@ -89,6 +89,10 @@ def save(self):
         raise Exception('Invalid path for oppia_tools library: %s' % pil_path)
     sys.path.insert(0, pil_path)
 
+OPPIA_TOOLS = [
```

Comment:
Why are you putting this here? Put it in the usual place, similar to all other third-party libs.

OPPIA_TOOLS stuff does not get uploaded to production.

Reply:
I've removed this.